lib/blueprinter/configuration.rb

```diff
@@ -1,9 +1,14 @@
 module Blueprinter
   class Configuration
-    attr_accessor :generator
+    attr_accessor :generator, :method
 
     def initialize
       @generator = JSON
+      @method = :generate
+    end
+
+    def jsonify(blob)
+      generator.send(method, blob)
```

Comment:
Thank you for doing this! Just one minor request. Can we use `public_send` over `send`?

Reply:
Thanks for getting back on this. I've changed it to use public_send. Let me know if you need anything else.